sigstore-java/src/main/java/dev/sigstore/tuf/Updater.java

```diff
@@ -100,13 +97,7 @@ Root updateRoot()
 
     // 5.3.2) load the trust metadata file (root.json), get version of root.json and the role
     // signature threshold value
-    Optional<Root> localRoot = localStore.loadTrustedRoot();
-    Root trustedRoot;
-    if (localRoot.isPresent()) {
-      trustedRoot = localRoot.get();
-    } else {
-      trustedRoot = GSON.get().fromJson(Files.readString(trustedRootPath), Root.class);
```

Comment:
It sounds super-strange to have a TUF store without a root. I believe `loadTrustedRoot` should return non-`Optional` value.

@patflynn ?

Reply:
The local TUF store can be empty until a successful update has occurred. A trusted root needs to be passed to the updater along with the remote url, but that trusted root is bundled with the client or provided by the user and doesn't typically get copied into the store.